docs/content/deploy.md

```diff
@@ -0,0 +1,121 @@
+1. Build and upload a game assembly for the deployment
+    - The assembly  includes executable files for the client-workers and server-workers, and the assets both types of workers use (such as the models and textures used by the client-server - that is, the game executable code - to visualise the game).
+    1. To build an assembly; in the Unity Editor, select **SpatialOS** > **Build all workers for cloud**.
+
+    2. To upload an assembly; open a terminal and navigate to the directory in which your game is in (the repository you’ve cloned). Run `spatial cloud upload <assembly name>`.
+    - The `<assembly name>` is a label you create so you can identify this assembly in the next step - for example you could call it `MyGDKAssembly`.
+    > **It’s finished uploading when:** You see `spatial upload <assembly name> succeeded` printed in your terminal output.
+1. Launch a cloud deployment
+    - In the same terminal window, run `spatial cloud launch <assembly name> cloud_launch.json <deployment name> --snapshot=snapshots/default.snapshot`
+    - This command defaults to deploying to clusters located in the US. So if you’re in Europe, add the `--cluster_region=eu` flag for better latency.
+    > **It’s done when:** You see `Successfully created deployment` printed in your Unity Editor console output.
+
+ - About the `spatial cloud launch` command
+    - `spatial cloud launch` deploys a project to the cloud. Its full syntax is:
+        `spatial cloud launch <assembly name> <launch configuration> <deployment name> --snapshot=<snapshot file>`
+        where:
+        - `<assembly name>` is the name of the assembly the deployment will use (the one you named above).
+        - `<launch configuration>` is the configuration file for the deployment.
+        - `<deployment name>` is the name you choose as you enter the command; , you’ll use this to identify the deployment. This must be in lowercase.
+        - `<snapshot file>` is the snapshot of the world you want to start from.  See the the GDK documentation for further information on [snapshots](./content/snapshots.md).
+
+1. Launch a game client:
+    1. Open the SpatialOS  [Console](https://console.improbable.io/projects). You’ll see the project and the deployment you just created.
+    1. In the SpatialOS Console, select the deployment’s name to open the overview page.
+    1. Select **Launch**.
+        > You can ignore the prompt to install the Launcher, as it’s installed as part of the SpatialOS Installer.
+    1. To get links to share the game with others, select **Share**.
+    1. Once you’ve finished playing, select **Stop** in the Console.
+
+
+## Get started with the Playground project
+
+This guide walks you through starting a SpatialOS game from the Unity Editor, either using SpatialOS running locally on your computer, or using SpatialOS in the cloud.
+It uses the `Playground` as an example which is a basic Unity project using SpatialOS.
```

Comment:
It uses the `Playground` as an example which is a basic Unity project using SpatialOS.
-->
It uses the `Playground` as an example; the `Playground` comes with the GDK. It is a basic Unity project which uses SpatialOS.